epoxy-processor/src/main/java/com/airbnb/epoxy/GeneratedModelWriter.java

```diff
@@ -623,6 +656,41 @@ private void addEqualsLineForType(Builder builder, boolean useObjectHashCode, Ty
     }
   }
 
+  private void addEqualsLineForTypeUsingGetter(Builder builder, boolean useObjectHashCode,
```

Comment:
To avoid duplicating hash/equals methods, what if we made a little method like `attributeGetterCode` and `attributeSetterCode`, which would so something like
```
if(isPrivateAttribute){
   return attribute.name + "()"; // Assuming we enforce setter name has to equal field name
} else {
   return attribute.name;
}

Setter might be trickier but maybe possible?